### PR TITLE
Added the device invetory related savedsearches

### PR DESCRIPTION
**Pull request by hardikhdholariya**
- Added the saved searches related to device inventory.
- Added the lookup for specific device and main lookup for all devices.

Note: The existing device inventory is kept as is for now. We'll remove it at the end and update the lookup names when convenient.

### CODE REVIEW

File: cyences_app_for_splunk/default/transforms.conf [hunk 1]
```diff
@@ -87,6 +87,53 @@ collection = cs_kaspersky_inventory_collection
 fields_list = _key, time, ip, hostname, kaspersky_collected_by, kaspersky_version, kaspersky_host, mac_address,kaspersky_status
 
 
+# Device Inventory V2
+[cs_device_inventory_test]
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/transforms.conf:91]: Use v2 postfix instead of test.

**Review by VatsalJagani — APPROVED** [review]

File: cyences_app_for_splunk/default/collections.conf [hunk 2]
```diff
@@ -47,6 +47,27 @@ accelerated_fields.incident_id = {"notable_event_id": 1}
 
 [cs_kaspersky_inventory_collection]
 
+
+# Device Inventory v2
+[cs_device_inventory_collection_test]
```
[the remark this answers is not in the document]
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/collections.conf]: Updated the test postfix with v2 everywhere.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -3501,6 +3501,314 @@ search = | inputlookup cs_device_inventory \
...
+| table time, product_name, product_uuid, ip, mac_address, lansweeper_id, hostname, lansweeper_collected_by, Site, AssetType, lansweeper_state, Domain, GroupName, OSVersion, BuildNumber, AssetVersion, lansweeper_user, lansweeper_os, Description, IPLocation, lansweeper_fqdn, antivirus, AssetDomain, FirstSeen, LastSeen, AssetName, Serialnumber, Processor, Model, Manufacturer, OSRelease, OSname, SystemVersion, Memory, LsAgentVersion, LastLsAgent, LastChanged, DNSName \
+| cyencesdevicemanager operation="addentries" \
+| stats values(GroupName) as GroupName, values(antivirus) as antivirus, values(Processor) as Processor, first(*) as * by lansweeper_id \
+| eval _key=lansweeper_id \
+| outputlookup cs_lansweeper_inventory_test
+# Note - We are not appending previous lookup in case of Lansweeper as Lansweeper Add-on always fetch full asset list.
```
[the remark this answers is not in the document]
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/savedsearches.conf]: Updated

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 4]
```diff
@@ -3501,6 +3501,314 @@ search = | inputlookup cs_device_inventory \
...
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_windows_defender` EventCode=1151 | `cs_windows_defender_inventory_fill_search`
+action.cyences_notable_event_action.products = Windows Defender
+
+
+[Device Inventory - CrowdStrike - V2]
```
[the remark this answers is not in the document]
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/savedsearches.conf:3665]: Yes, it's tested on cerb instance.

**Review by VatsalJagani — APPROVED** [review]

File: cyences_app_for_splunk/default/macros.conf [hunk 5]
```diff
@@ -194,13 +194,16 @@ iseval = 0
 
 [cs_windows_defender_inventory_fill_search]
 definition = dedup host \
-| eval hostname=lower(host), ip="", mac_address="", Last_full_scan_start_time=if(Last_full_scan_start_time="1/1/1601 12:00:00 AM", "-", Last_full_scan_start_time), Last_full_scan_end_time=if(Last_full_scan_end_time="1/1/1601 12:00:00 AM", "-", Last_full_scan_end_time),  Last_quick_scan_start_time=if(Last_quick_scan_start_time="1/1/1601 12:00:00 AM", "-", Last_quick_scan_start_time), Last_quick_scan_end_time=if(Last_quick_scan_end_time="1/1/1601 12:00:00 AM", "-", Last_quick_scan_end_time) \
+| eval hostname=lower(host), ip="", mac_address="", product_name="Windows Defender", product_uuid=host, Last_full_scan_start_time=if(Last_full_scan_start_time="1/1/1601 12:00:00 AM", "-", Last_full_scan_start_time), Last_full_scan_end_time=if(Last_full_scan_end_time="1/1/1601 12:00:00 AM", "-", Last_full_scan_end_time), Last_quick_scan_start_time=if(Last_quick_scan_start_time="1/1/1601 12:00:00 AM", "-", Last_quick_scan_start_time), Last_quick_scan_end_time=if(Last_quick_scan_end_time="1/1/1601 12:00:00 AM", "-", Last_quick_scan_end_time), time=_time \
```
Comment (mahirchavda) [inline cyences_app_for_splunk/default/macros.conf:197]: product_uuid = lower(host) ?

**Review by mahirchavda — APPROVED** [review]

**Pull request merged**